### PR TITLE
Bootstrap color fix/bd

### PR DESCRIPTION
**Pull request by dhunganabibek**
### Type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Checklist

- [ ] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [ ] My code follows the code style of the project _(using `npm run lint`)_
- [ ] My change introduces changes to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

#### Live previews

- <https://deploy-preview-{your_pr_number}--twbs-bootstrap.netlify.app/>

### CODE REVIEW

File: site/content/docs/5.3/customize/color.md [hunk 1]
```diff
@@ -365,19 +365,36 @@ Be sure to monitor contrast ratios as you customize colors. As shown below, we'v
...
+          </div>
+        </div>
+      </div>
       {{ end }}
     </div>
     {{ end -}}
   {{ end -}}
 
-  <div class="col-md-4 mb-3">
+  <div class="col-md-4 mb-3 border">
```
Comment (abrahamguo) [inline site/content/docs/5.3/customize/color.md]: why?

File: site/content/docs/5.3/customize/color.md [hunk 2]
```diff
@@ -365,19 +365,36 @@ Be sure to monitor contrast ratios as you customize colors. As shown below, we'v
         {{ $color.hex }}
       </div>
       {{ range slice 50 | append (seq 100 100 900) }}
-      <div class="p-3 bd-{{ $color.name }}-{{ . }}">${{ $color.name }}-{{ . }}</div>
+      <div class="p-3 bd-{{ $color.name }}-{{ . }}">
+        <div class="d-flex flex-column">
+          <span>${{ $color.name }}-{{ . }}</span>
+          <div style="font-size: 0.75rem" class="d-flex flex-column">
```
Comment (abrahamguo) [inline site/content/docs/5.3/customize/color.md]: why is d-flex needed?

File: site/content/docs/5.3/customize/color.md [hunk 3]
```diff
@@ -365,19 +365,36 @@ Be sure to monitor contrast ratios as you customize colors. As shown below, we'v
         {{ $color.hex }}
       </div>
       {{ range slice 50 | append (seq 100 100 900) }}
-      <div class="p-3 bd-{{ $color.name }}-{{ . }}">${{ $color.name }}-{{ . }}</div>
+      <div class="p-3 bd-{{ $color.name }}-{{ . }}">
+        <div class="d-flex flex-column">
```
Comment (abrahamguo) [inline site/content/docs/5.3/customize/color.md]: why is d-flex needed?

File: site/content/docs/5.3/customize/color.md (hunk 2 above)
Comment (dhunganabibek) [reply site/content/docs/5.3/customize/color.md]: I was using span, I can remove d-flex by changing span to div.

File: site/content/docs/5.3/customize/color.md [hunk 4]
```diff
@@ -377,7 +385,16 @@ Be sure to monitor contrast ratios as you customize colors. As shown below, we'v
...
-    <div class="p-3 bd-gray-{{ .name }}">$gray-{{ .name }}</div>
+    <div class="p-3 bd-gray-{{ .name }}">
+      <div class="flex">
+        $gray-{{ .name }}
+         <div style="font-size: 0.75rem" >
+            <div>.bg-gray-{{ .name }}</div>
+            <div>.text-gray-{{ .name }}</div>
+         </div>
+      </div>
+    </div>
```
Comment (abrahamguo) [inline site/content/docs/5.3/customize/color.md:396]: as discussed

File: site/content/docs/5.3/customize/color.md [hunk 5]
```diff
@@ -377,7 +385,16 @@ Be sure to monitor contrast ratios as you customize colors. As shown below, we'v
       #777
     </div>
   {{- range $.Site.Data.grays }}
-    <div class="p-3 bd-gray-{{ .name }}">$gray-{{ .name }}</div>
+    <div class="p-3 bd-gray-{{ .name }}">
+      <div>
+        $gray-{{ .name }}
+         <div style="font-size: 0.75rem" >
```
Comment (abrahamguo) [inline site/content/docs/5.3/customize/color.md:391]: remove space

**Review by abrahamguo — APPROVED** [review]

**Pull request merged**